frontend/src/components/organizations/NotForProfitQuestions.vue

```diff
@@ -0,0 +1,213 @@
+              <AppLabel>Are the majority of the elected board members full time BC residents?</AppLabel>
+            </v-col>
+          </v-row>
+          <v-row no-gutters>
+            <v-col>
+              <v-radio-group v-model="model.boardMembersResidentsOfBC" :readonly="false" :rules="[...rules.required]">
+                <v-row no-gutters>
+                  <v-col cols="12" sm="2" lg="1">
+                    <v-radio label="Yes" :value="1"></v-radio>
+                  </v-col>
+                  <v-col cols="12" sm="2" lg="1">
+                    <v-radio label="No" :value="0"></v-radio>
+                  </v-col>
+                </v-row>
+              </v-radio-group>
+            </v-col>
+          </v-row>
+        </v-card>
+      </v-container>
+    </v-card>
+  </v-skeleton-loader>
+</template>
+
+<script>
+import AppLabel from '@/components/ui/AppLabel.vue'
+import AppDateInput from '@/components/ui/AppDateInput.vue'
+import alertMixin from '@/mixins/alertMixin'
+import rules from '@/utils/rules'
+import AppDocumentUpload from '@/components/ui/AppDocumentUpload.vue'
+import moment from 'moment'
+import { DOCUMENT_TYPES } from '@/utils/constants'
+import { isEmpty, cloneDeep } from 'lodash'
+
+export default {
+  components: { AppLabel, AppDateInput, AppDocumentUpload },
+  mixins: [alertMixin],
+  props: {
+    organization: {
+      type: Object,
+      required: true,
+      default: () => {
+        return {}
+      },
+    },
+    loading: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  emits: ['update', 'deleteDocument', 'documentsComplete'],
+  data() {
+    return {
+      choice: undefined,
+      selectedDate: undefined,
+      model: undefined,
+      todayDate: new Date().toISOString().slice(0, 10),
+    }
+  },
+  computed: {
+    isReferenceLetterRequired() {
+      const pastDate = moment().subtract(4, 'years').startOf('day')
+      const modelDate = moment(this.model?.dateOfIncorporation).startOf('day')
+      return pastDate < modelDate
+    },
+    areDocumentsComplete() {
+      if (this.isReferenceLetterRequired) {
+        return !isEmpty(this.model.documentsToUpload) || !isEmpty(this.model.uploadedDocuments)
+      }
+      return true
+    },
+  },
+  watch: {
+    model: {
+      handler(value) {
+        this.$emit('update', value)
+        this.$emit('documentsComplete', this.areDocumentsComplete)
+      },
+      deep: true,
+    },
+  },
+  async created() {
+    this.rules = rules
+    this.DOCUMENT_TYPES = DOCUMENT_TYPES
+    this.model = cloneDeep(this.organization)
+  },
+  methods: {
+    deleteUploadedDocument(documentId) {
+      const foundDoc = this.model.uploadedDocuments.find((el) => el.documentId === documentId)
+      this.model.uploadedDocuments.splice(this.model.uploadedDocuments.indexOf(foundDoc), 1)
+      this.$emit('update', this.model)
+      this.$emit('deleteDocument', documentId)
+    },
+  },
+}
+</script>
+
+<style scoped>
+.card-outline {
```

Comment:
can we use "soft-outline" in main.css? we should also update OrganizationInfo to use "soft-outline".

Reply:
done!